src/js/stores/AppFormStore.js

```diff
@@ -553,8 +554,12 @@ function executeAction(action, setFieldFunction) {
   if (!errorOccurred) {
     rebuildModelFromFields(storeData.app, storeData.fields, fieldId);
     AppFormStore.emit(FormEvents.CHANGE, fieldId);
+
+    AppFormStore.emit(FormEvents.FIELD_VALIDATION_ERROR);
```

Comment:
I think it was because otherwise the PluginDispatcher won't be triggered, thus the Plugins don't update.